app/controllers/papers_controller.rb

```diff
@@ -40,11 +42,18 @@ def update
       paper.update(update_paper_params)
     end
 
+    notify_paper_edited! if params[:paper][:locked_by_id].present?
+
     respond_with paper
   end
 
   # non RESTful routes
 
+  def activity_feed
+    activity_feeds = ActivityFeed.where(feed_name: 'manuscript', subject_id: paper.id).order('created_at DESC')
+    respond_with activity_feeds, each_serializer: ActivityFeedSerializer, root: 'feeds'
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.